examples/knex/mysql2/connect.cjs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {Connector} = require('@google-cloud/cloud-sql-connector');
+const knex = require('knex');
+
+async function connect({ instanceConnectionName, user, databaseName }) {
```

Comment:
In the test you are passing argument `db` and here it is called `databaseName`. Also, isn't expected property name in connectionOptions called `database`?

Reply:
good catch! fixing it here: https://github.com/GoogleCloudPlatform/cloud-sql-nodejs-connector/pull/264